packages/apollo-server-azure-functions/src/azureFunctions.ts

```diff
@@ -1,153 +0,0 @@
-import { ValueOrPromise } from 'apollo-server-env';
-
-export type Context = {
-  invocationId: string;
-  bindingData: any;
-  bindings: any;
-
-  log: (text: any) => void;
-  done: (err?: any, output?: object) => void;
-};
-
-export type HttpMethod =
-  | 'GET'
-  | 'POST'
-  | 'PUT'
-  | 'DELETE'
-  | 'HEAD'
-  | 'OPTIONS'
-  | 'TRACE'
-  | 'CONNECT'
-  | 'PATCH';
-
-export type FunctionRequest = {
-  originalUrl: string;
```

Comment:
We currently rely on this `originalUrl` property, which the new typings on `@azure/functions` seem to claim is no longer available, while again, the [documentation for the request object](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node#request-object) seem to claim is still necessary?

Reply:
🤦‍♂️ this is my bad. Since I had `1.0.1-beta2` installed, it was `originalUrl`. I actually made that change in my test project, but didn't do the project link this time to make it quicker. 

I've now changed this to `url` based on the current package available